config/webpack.dev.js

```diff
@@ -26,7 +28,7 @@ const METADATA = webpackMerge(commonConfig.metadata, {
   HMR: HMR
 });
 
-const frontendConfig = commonConfig.frontendConfig;
+const frontendConfig = _.merge(commonConfig.frontendConfig, { apiEndpoint: 'http://localhost:8080/v1' });
```

Comment:
`Globals` could be used instead I guess, More of a `Defaults` than `Globals` perhaps.